lib/dialogs/share.jsx

```diff
@@ -111,7 +112,7 @@ export default React.createClass( {
 		const { note } = this.props.params;
 		const data = note && note.data || {};
 		const isPublished = includes( data.systemTags, 'published' );
-		const publishURL = isPublished && data.publishURL !== '' && data.publishURL && `http://simp.ly/publish/${data.publishURL}` || null;
+		const publishURL = ! isEmpty( data.publishURL ) && `http://simp.ly/p/${data.publishURL}` || null;
```

Comment:
Thanks for simplifying. I guess I meant that the `&&` `||` with `null` was funny. Though I remember seeing the use in `value`, I though there was already protection against this.

Might still consider using a ternary here or in the view below since it does exactly what this line is doing…

```js
const publishURL = ! isEmpty( data.publishURL )
	? `http://simp.ly/p/${data.publishURL}`
	: '';
```

or handle it in the view
```js
const publishURL = ! isEmpty( data.publishURL ) && `http://simp.ly/p/${data.publishURL}`;

…

// looks kinda ugly, but somewhere something is going to be ugly anyway ¯\_(ツ)_/¯
value={ publishURL && publishURL }
```